src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1519,4 +1519,13 @@
     <value>and all other platforms</value>
     <comment>This call site is reachable on: 'windows' 10.0.2000 and later, and all other platforms</comment>
   </data>
+  <data name="UseAssignableTypeForForeachVariableDescription" xml:space="preserve">
+    <value>There is no implicit conversion between collection element and iteratoi variable, this will result in runtime exception.</value>
```

Comment:
Small typo. Note that you will have to regenerate the resources after applying this fix.

```suggestion
    <value>There is no implicit conversion between collection element and iterator variable, this will result in runtime exception.</value>
```